### ISSUE TITLE
Can not connect

### ISSUE THREAD

**Issue by bshkrv** [dino/dino#96]
After entering account details and toggling 'Connect' button, I keep getting connected and disconnected.

Dino logs one of the following error  messages:
```
Stream Error: NS name ack not found.
```
or
```
Stream Error: mandatory-to-negotiate feature not negotiated
```

**Comment by IamPr377y** [comment]
It happens on different networks, even on the same local network with the server.

**Comment by mar-v-in** [comment]
Which server or server software is that?

**Comment by IamPr377y** [comment]
@mar-v-in They say it's jabberd2.

**Comment by fiaxh** [comment]
Fixed in 6904bda
